test_project/rename_forward/urls.py

```diff
@@ -8,6 +8,7 @@
 class LinkedDataView(autocomplete.Select2QuerySetView):
     def get_queryset(self):
         qs = super(LinkedDataView, self).get_queryset()
+        print(self.forwarded.get('checkbox', None))
```

Comment:
Yeah, I'm using it for my django project all the time as it's the part of python mode in spacemacs.

I submited this along with this cause I assumed it is better to have some test, than none, even if it would be thrown away in a momend, and pdb/ipdb would stop the execution.